WcaOnRails/app/models/user.rb

```diff
@@ -293,6 +285,44 @@ def after_confirmation
     end
   end
 
+  def software_team?
+    team_member?('software') != false
```

Comment:
Remember that we're using it to check .team_leader at
https://github.com/pedrosino/worldcubeassociation.org/blob/teams/WcaOnRails/app/models/user.rb#L322-L324
?

On Mon, Mar 21, 2016 at 4:53 AM, Jeremy Fleischman <notifications@github.com
> wrote:

> In WcaOnRails/app/models/user.rb
> <https://github.com/cubing/worldcubeassociation.org/pull/390#discussion_r56789044>
> :
>
> > @@ -293,6 +285,44 @@ def after_confirmation
> >      end
> >    end
> >
> > +  def software_team?
> > +    team_member?('software') != false
>
> This looks like a mess. Why isn't team_member? returning true or false?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/390/files/4cfc692de289d2d4d24e03be7e8db4b02da2a89d#r56789044>
>